firmware/tunerstudio/tunerstudio.template.ini

```diff
@@ -2849,20 +2847,34 @@ dialog = launch_control_stateDialog, "launch_control_state"
 		field = "TPS minimum valid value",	tpsErrorDetectionTooLow, {tps1_1AdcChannel != @@ADC_CHANNEL_NONE@@}
 		field = "TPS maximum valid value",	tpsErrorDetectionTooHigh, {tps1_1AdcChannel != @@ADC_CHANNEL_NONE@@}
 
+	dialog = tpsRedundancy, "TPS Redundancy"
+		field = "!https://wiki.fome.tech/Advanced-Features/ETB/Redundant-Sensors/"
```

Comment:
I don't know how to generate a "short link" for the wiki, I see other links like `wiki.fome.tech/r/tps`, not sure we need/want that here